packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -1176,11 +1191,15 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
   @override
   Widget build(BuildContext context) {
     assert(debugCheckHasOverlay(context));
+    final SelectionRegistrar? registrar = SelectionContainer.maybeOf(context);
```

Comment:
I don't think we should be stacking selectable regions. Is this for setting different separate for the subtree? if so, can you create a new named SelectionContainer constructor that takes in a separator and create a `_SelectableRegionContainerDelegate` with that separate in its state?

for example something like `SelectionContainer.options(separator: '\n');`

Reply:
How does making a different constructor change anything?

Reply:
I'm not sure what to do here. If the SelectableRegions don't share a registrar how will they get separated when copied?